src/pages/Bid/BidPerchase.jsx

```diff
@@ -0,0 +1,276 @@
+
+        <ShipmentInfo>
+          <ShipmentAddressDetailBox>
+            <div className="ShipmentTitle">배송 주소</div>
+            <div className="ShipmentReciever">이경민</div>
+            <div className="ShipmentAddress">
+              (06325)서울 강남구 선릉로8길 22 (개포동) 102-203 010-2621-0847
+            </div>
+          </ShipmentAddressDetailBox>
+          <ShipmentInfoChangeButtonBox>
+            <button>변경</button>
+          </ShipmentInfoChangeButtonBox>
+        </ShipmentInfo>
+
+        <SubmitButtonBox>
+          <ContinueBidButton type="submit">구매 입찰 계속</ContinueBidButton>
+        </SubmitButtonBox>
+      </BidBox>
+    </BidMain>
+  );
+}
+
+const BidMain = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-top: 98px;
+  border: solid;
+  background-color: green;
+`;
+
+const ProductImgBox = styled.div`
+  height: 550px;
+  padding-bottom: 20px;
+  margin-top: 79px;
+  margin-left: 50px;
+  background-color: white;
+  box-shadow: 2.5px 2.5px 10px 2.5px ${({ theme }) => theme.subFontColor};
+`;
+
+const ProductImgFrame = styled.div`
+  width: 450px;
+  height: 100%; //  ImgInfo 를 감쌀수 있어야 한다
+  margin: 20px;
+  padding: 10px;
+  background-color: blue;
+`;
+
+const ProductImg = styled.img`
+  width: 430px;
+  height: 430px;
+  border-radius: 15px;
+`;
+
+const ImgInfo = styled.div`
+  background-color: yellow;
+`;
+
+const BidBox = styled.div`
+  //flex: 0 0 20%;
+  width: 50%;
+
+  margin: 10px;
+  margin-right: 50px;
+  margin-top: 79px;
+  flex-direction: column;
+  ${({ theme }) => theme.setFlex('flex-start')};
+  background-color: blue;
+`;
+
+const BidDetail = styled.div`
+  background-color: #ffffff;
+  width: 100%;
+`;
+
+//피알올리기  - 이 부분 너비가 BidBox 범위를 넘어가는데 이유를 모르겠습니다
+const SuggestPrice = styled.div`
+  display: flex;
+  justify-content: center;
+  width: 100%;
+`;
+
+const SelectBidBox = styled.div`
+  padding: 0 35px;
+  display: flex;
+  justify-content: space-between;
+`;
+
+const ImmediatePerchasePriceBox = styled.div`
+  margin: 30px;
+`;
+
+const ImmediatePerchasePriceTitle = styled.div`
+  width: 300px;
+  text-align: center;
+  font-size: 13px;
+  margin: 5px;
+  padding-left: 100px;
+`;
+
+const ImmediatePerchasePrice = styled.div`
+  text-align: center;
+  padding-left: 100px;
+`;
+
+const ImmediateSalePriceBox = styled.div`
+  margin: 30px;
+`;
+
+const ImmediateSalePriceTitle = styled.div`
+  width: 300px;
+  text-align: center;
+  font-size: 13px;
+  margin: 5px;
+  padding-right: 100px;
+`;
+
+const ImmediateSalePrice = styled.div`
+  text-align: center;
+  padding-right: 100px;
+`;
+
+const SelectPerchaseOrSaleButton = styled.div`
+  ${({ theme }) => theme.setFlex('center')};
+  width: 100%;
+`;
+
+const BidPerchaseButton = styled.button`
+  width: 50%;
+  height: 40px;
+  border-radius: 18px;
+  background-color: ${props =>
+    props.backgroundColor
+      ? ({ theme }) => theme.red
+      : ({ theme }) => theme.cardBackground}; // 아예적용이 안되고 있는듯 ?
```

Comment:
- 지금 코드를 보시면 삼항연산자의 조건에 따라 **함수를** 리턴하고 있습니다. `({ theme }) => theme.red` 혹은 `({ theme }) => theme.cardBackground` 말이죠!
- 스타일 컴포넌트가 작동하기 위해선 스트링을 반환해야 합니다! 따라서 아래와 같이 작성해주셔야 합니다!
```suggestion
  background-color: ${props =>
    props.backgroundColor
      ? props.theme.red
      : props.theme.cardBackground};
```

Reply:
수정했습니다